src/ontology/hp-edit.owl

```diff
@@ -168662,7 +168660,7 @@ AnnotationAssertion(dc:date <http://purl.obolibrary.org/obo/HP_0032368> "2019-02
 AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0032368> <https://orcid.org/0000-0002-0736-9199>)
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0032368> "When describing acid-base abnormalities, acidosis and alkalosis refer to physiological processes that lower or raise the pH, respectively, while acidemia and alkalemia simply describe the state of an abnormal blood pH. This distinction is important: a patient with acidemia could have both a respiratory and metabolic acidosis as well as a concurrent metabolic alkalosis.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0032368> "Acidemia")
-EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0032368> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000470> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CHEBI_24431> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0001977>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000087> <http://purl.obolibrary.org/obo/CHEBI_37527>))) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
+EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0032368> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0001844> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0000178>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
```

Comment:
Should acidemia be defined in terms of "decreased acidity"? (I usually do not look at content, but since I want this to be merged asap :P)

Reply:
It is kind of confusing, because increased acidity means [lower pH](https://en.wikipedia.org/wiki/PH), but 'Acidemia' is [An abnormally low blood pH (usually defined as less than 7.35).](https://www.ebi.ac.uk/ols4/ontologies/hp/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FHP_0032368)

'pH' is indeed a synonym of [PATO:0001842 acidity](https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0001842), but the textual definition of [PATO:0001844 increased acidity](https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0001844?lang=en)) refers to higher acidity. People who measure acidity and pH in laboratories and report related phenotypes are aware of the negative correlation between acidity and pH, and when they say 'increased acidity' it should be unambiguous what it means in terms of pH.
In summary, acidemia is correctly defined as increased acidity here.